--- qemu.conf.original	2016-09-24 13:12:45.198671353 +0000
+++ qemu.conf	2016-09-24 13:22:25.976659916 +0000
@@ -601,11 +601,9 @@
 # differently for different guest architectures. For instance, it's OVMF
 # for x86_64 and i686, but it's AAVMF for aarch64. The libvirt default
 # follows this scheme.
-#nvram = [
-#   "/usr/share/OVMF/OVMF_CODE.fd:/usr/share/OVMF/OVMF_VARS.fd",
-#   "/usr/share/OVMF/OVMF_CODE.secboot.fd:/usr/share/OVMF/OVMF_VARS.fd",
-#   "/usr/share/AAVMF/AAVMF_CODE.fd:/usr/share/AAVMF/AAVMF_VARS.fd"
-#]
+nvram = [
+   "/usr/share/edk2-ovmf/OVMF_CODE.fd:/usr/share/edk2-ovmf/OVMF_VARS.fd" 
+]
 
 # The backend to use for handling stdout/stderr output from
 # QEMU processes.
